src/plugins/explore/public/components/patterns_table/utils/constants.tsx

```diff
@@ -23,14 +26,18 @@ export const patternsTableColumns: Array<EuiBasicTableColumn<PatternItem>> = [
       }
       return `${(val * 100).toFixed(2)}%`;
     },
-    width: '15%',
+    width: '10%',
   },
   {
-    field: 'pattern',
-    name: i18n.translate('explore.patterns.table.column.pattern', {
-      defaultMessage: 'Pattern',
+    field: 'sample',
+    name: i18n.translate('explore.patterns.table.column.sampleLog', {
+      defaultMessage: 'Pattern Sample Log',
     }),
-    render: (pattern: string) => pattern || '—',
+    render: (sample: string) => {
+      const sanitizedSampleLog = dompurify.sanitize(sample);
+      // eslint-disable-next-line react/no-danger
+      return <span dangerouslySetInnerHTML={{ __html: sanitizedSampleLog || '—' }} />;
```

Comment:
why set inner html?

Reply:
The sample log, using the helper method below called `highlightLogUsingPattern`, will include <mark> tags in the real log that we pull, for the purpose of highlighting all of the dynamic vs static elements. This is similar to the existing functionality where DQL search terms will highlight terms in the results, and so I'm utilizing the same method done there. The dangerouslySetInnerHtml is needed to have the <mark> tags be recognized as HTML.

Reply:
nit: it is a little bit weird that constants file is not a file of constants.

Reply:
yeah this should be moved out of constants.

i don't think its nit. we may have checks in the future where we ignore `constants.ts` in testing coverage, but this has functional logic in it, so we don't want it ignored